src/argilla/server/database.py

```diff
@@ -43,9 +49,50 @@ def get_db():
         db.close()
 
 
-def migrate_db():
-    alembic_config = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "alembic.ini"))
-    alembic.config.main(argv=["-c", alembic_config, "upgrade", "head"])
+TAGGED_REVISIONS = OrderedDict(
+    {
+        "1.7": "1769ee58fbb4",
+        "1.8": "ae5522b4c674",
+    }
+)
+
+
+def migrate_db(revision: Optional[str] = None):
+    alembic_config_file = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", "alembic.ini"))
+    if revision:
+        revision = TAGGED_REVISIONS.get(revision, revision)
+
+        current_revision = _get_current_revision(alembic_config_file)
+        script = ScriptDirectory.from_config(Config(alembic_config_file))
+
+        try:
+            script.walk_revisions(base=current_revision, head=revision)
+            action = "upgrade"
+        except CommandError:
+            action = "downgrade"
+
+    else:
+        revision = "head"
+        action = "upgrade"
+
```

Comment:
Agree. This can be included as future command improvements:

- `--verbose` showing more info about the applied migrations
- `--wait-confirmation`  or `--confirm` to force user confirmation

Reply:
Finally, I put the alembic command by default. It can help to detect migration problems.